fluentd-loggly/td-agent.conf

```diff
@@ -324,13 +324,30 @@
   reserve_data true
 </filter>
 
-# Send logs to Loggly
+# Send logs to Loggly and Elastic
 <match **>
-  type loggly_buffered
-  loggly_url "https://logs-01.loggly.com/bulk/#{ENV['LOGGLY_TOKEN']}/tag/fluentd_bulk"
-  output_include_time true
-  time_precision_digits 3
-  buffer_type file
-  buffer_path /var/log/fluentd_loggly_buffer
-  flush_interval 10s
+  @type copy
+  <store>
+    @type loggly_buffered
+    loggly_url "https://logs-01.loggly.com/bulk/#{ENV['LOGGLY_TOKEN']}/tag/fluentd_bulk"
+    output_include_time true
+    time_precision_digits 3
+    buffer_type file
+    buffer_path /var/log/fluentd_loggly_buffer
+    flush_interval 10s
+  </store>
+  <store>
+    @type elasticsearch
+    host "#{ENV['ELASTIC_HOST']}"
+    scheme "#{ENV['ELASTIC_SCHEME']}"
+    port "#{ENV['ELASTIC_PORT']}"
+    user "#{ENV['ELASTIC_USERNAME']}"
+    password "#{ENV['ELASTIC_PASSWORD']}"
```

Comment:
Are these Kubernetes secrets?

Reply:
No, its just part of the configuration for now